### PR TITLE
Added `clear_raw()` methods to Encoder.

### PR DESCRIPTION
**Pull request by icefoxen**
For color, depth and stencil views.  Also re-wrote the
non-raw versions to just use the raw ones.

Fixes #issue
PR checklist:
- [x] `cargo build` succeeds
- [x] `cargo test` succeeds
- [x] tested examples with the following backends: GL

### CODE REVIEW

File: src/render/src/encoder.rs [hunk 1]
```diff
@@ -510,20 +510,41 @@ impl<R: Resources, C: command::Buffer<R>> Encoder<R, C> {
...
     /// Clear a stencil view with a specified value.
     pub fn clear_stencil<T: format::StencilFormat>(&mut self,
                          view: &handle::DepthStencilView<R, T>, stencil: Stencil) {
-        let target = self.handles.ref_dsv(view.raw()).clone();
+        self.clear_stencil_raw::<T>(view.raw(), stencil);
+    }
+
+
+    /// Clears the supplied `RawRenderTargetView` to the supplied `ClearColor`.
+    pub fn clear_raw<T: format::RenderFormat>(&mut self,
```
Comment (kvark) [inline src/render/src/encoder.rs]: let's just accept a `ClearColor` and no bother with passing `T` in (which is not expected for a `_raw` method)

File: src/render/src/encoder.rs [hunk 2]
```diff
@@ -510,20 +510,41 @@ impl<R: Resources, C: command::Buffer<R>> Encoder<R, C> {
...
+                 view: &handle::RawRenderTargetView<R>, value: T::View)
+    where T::View: Into<command::ClearColor> {
+        let target = self.handles.ref_rtv(view).clone();
+        self.command_buffer.clear_color(target, value.into())
+    }
+
+
+    /// Clear a raw depth view with a specified value.
+    pub fn clear_depth_raw<T: format::DepthFormat>(&mut self,
```
Comment (kvark) [inline src/render/src/encoder.rs]: similarly, let's remove `T` argument

File: src/render/src/encoder.rs [hunk 3]
```diff
@@ -510,20 +510,41 @@ impl<R: Resources, C: command::Buffer<R>> Encoder<R, C> {
...
+                       view: &handle::RawDepthStencilView<R>, depth: Depth) {
+        let target = self.handles.ref_dsv(view).clone();
+        self.command_buffer.clear_depth_stencil(target, Some(depth), None)
+    }
+
+    /// Clear a raw stencil view with a specified value.
+    pub fn clear_stencil_raw<T: format::StencilFormat>(&mut self,
```
Comment (kvark) [inline src/render/src/encoder.rs]: aaand here

**Comment by icefoxen** [comment]
Easily done!  Thank you.

File: src/render/src/encoder.rs [hunk 4]
```diff
@@ -510,39 +510,38 @@ impl<R: Resources, C: command::Buffer<R>> Encoder<R, C> {
...
+        self.clear_stencil_raw(view.raw(), stencil);
     }
 
 
     /// Clears the supplied `RawRenderTargetView` to the supplied `ClearColor`.
-    pub fn clear_raw<T: format::RenderFormat>(&mut self,
-                 view: &handle::RawRenderTargetView<R>, value: T::View)
-    where T::View: Into<command::ClearColor> {
+    pub fn clear_raw(&mut self,
+                 view: &handle::RawRenderTargetView<R>, value: command::ClearColor) {
```
Comment (kvark) [inline src/render/src/encoder.rs]: nit indentation

**Comment by kvark** [comment]
@icefoxen ping

**Comment by icefoxen** [comment]
Squashed in #2197

**Pull request closed without merging**